spec/models/preserved_copy_spec.rb

```diff
@@ -156,35 +103,24 @@
   end
 
   describe '#update_status' do
-    let(:pc) do
-      # using create here, because if the object has never been saved, #changed? will always return true
-      PreservedCopy.create(
-        preserved_object_id: preserved_object.id,
-        endpoint_id: endpoint.id,
-        version: 0,
-        status: PreservedCopy::INVALID_MOAB_STATUS,
-        size: 1
-      )
-    end
-
     it 'does nothing if the status has not changed' do
       ran_the_block = false
-      pc.update_status(PreservedCopy::INVALID_MOAB_STATUS) { ran_the_block = true }
-      expect(pc.status).to eq PreservedCopy::INVALID_MOAB_STATUS
+      preserved_copy.update_status(preserved_copy.status) { ran_the_block = true }
+      expect(preserved_copy.status).to eq 'validity_unknown'
       expect(ran_the_block).to eq false
-      expect(pc.changed?).to eq false
+      expect(preserved_copy.status_changed?).to eq false
     end
 
     it 'runs the block and updates the status if the status has changed' do
       ran_the_block = false
-      pc.update_status(PreservedCopy::OK_STATUS) { ran_the_block = true }
-      expect(pc.status).to eq PreservedCopy::OK_STATUS
+      preserved_copy.update_status('ok') { ran_the_block = true }
+      expect(preserved_copy.status).to eq 'ok'
       expect(ran_the_block).to eq true
-      expect(pc.changed?).to eq true
+      expect(preserved_copy.status_changed?).to eq true
     end
   end
 
-  context '#matches_po_current_version?' do
+  describe '#matches_po_current_version?' do
```

Comment:
I was wondering this too -- I assumed they were synonymous.

Reply:
@ndushay You are right, it is functionally the same.  But is stylistically different with some convention around use.  Here we are talking about a part of the class, namely a method.  It is the thing being described and tested, a specialization of the main description (that is about the class).  

Context would be more like "when the user is logged in" or "when the submitted data is invalid".  It isn't the code being tested, but a condition (that might apply to many parts).  There is a rubocop-rspec rule that reflects this convention, requiring a `context` label to start with "if/when/with".

To my eye, `#matches_po_current_version?` isn't a context, because it is the thing being described/tested.  It isn't a condition that sometimes applies, sometimes doesn't.  Does that make sense?